test.js

```diff
@@ -1,57 +1,56 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+let stream;
+let output;
```

Comment:
This won't work as the tests are run concurrently, so they would be overridden out of order. Use `t.context` in `test.beforeEach` and the test for this purpose. 

https://github.com/avajs/ava#before--after-hooks